### ISSUE TITLE
Add learning rate monitoring to LighterLogger

### ISSUE THREAD

**Issue by ibro45** [project-lighter/lighter#18]
## Description

Mimick PL [LearningRateMonitor](https://pytorch-lightning.readthedocs.io/en/stable/api/pytorch_lightning.callbacks.LearningRateMonitor.html)

Implementation should be simple - access optimizers and schedulers of the `LighterSystem` using `pl_module` that's automatically passed to callbacks by PL and log them. Similar to https://github.com/project-lighter/lighter/blob/08cb305e965b9434411c59b752db40c454a5dc40/lighter/callbacks/logger.py#L237
